Replace "WASM" by "Wasm"

backes opened the pull request:
See the first sentence in the spec:
WebAssembly (abbreviated Wasm [1]), is [...]
--
[1] A contraction of “WebAssembly”, not an acronym, hence not using
all-caps.

marcoms commented:
Hi @backes, thanks for this contribution! We'd love to integrate these changes. Can I first check if you've signed our [CLA](https://pspdfkit.com/guides/web/current/miscellaneous/contributing/)?

backes committed:
Replace "WASM" by "Wasm"

See the first sentence in the spec:
WebAssembly (abbreviated Wasm [1]), is [...]
--
[1] A contraction of “WebAssembly”, not an acronym, hence not using
all-caps.

backes commented:
This would be covered by MIT license, right?
Let me check whether I can sign that CLA.

marcoms commented:
Yes this would be MIT licensed 👍

backes commented:
I am sorry, I cannot sign the CLA since our legal team thinks that there is too much ambiguity in your LICENSE file, and it's not clear which parts of the code (and my very minor contribution) would be covered by an open source license vs. your commercial license.

I will close this PR, and open an issue instead. And I hereby give up any copyright or ownership on the changes I proposed here, in case this would cause problems otherwise.
